Tests/SWBCoreTests/PIFLoadingTests.swift

```diff
@@ -683,18 +683,14 @@ private final class ProjectModelItemClass: ProjectModelItem {
         #expect(fileGroup.children.count == 2)
 
         // Examine its children
-        do
-        {
-            let fileRef = fileGroup.children[0] as! FileReference
+        if let fileRef = try? #require(fileGroup.children.first as? FileReference) {
```

Comment:
Unfortunately this pattern introduced some warnings and also unintentionally changed behavior -- since there is no else case, a previous crash is now an unexpected pass rather than a failure. Fixing in #135